Source/ARTTypes.m

```diff
@@ -387,20 +387,21 @@ - (nullable NSData *)art_archive {
 #endif
 }
 
-+ (nullable id)art_unarchive:(NSData *)data {
++ (nullable id)art_unarchiveFromData:(NSData *)data {
+    NSSet* allowedTypes = [NSSet setWithArray:@[ [NSArray class], [NSDictionary class], [self class]]];
```

Comment:
I think that all the places in this PR where you've written `[self class]` can just be `self`, since they're all inside class methods so `self` is already an instance of `Class`.

Reply:
Fixed 40b7a64